Class/D-Arrays-Recap/exercise2.js

```diff
@@ -6,14 +6,20 @@ let agesList = [24, 18, 15, 38, 17, 60, 21];
 // It should have one input parameter called age
 // It should return true if the age is 17 or above
 // otherwise return false
-
+function canDrive(age){
+    if(age >= 17){
+        return true;
+    }else{
+        return false;
+    }
+}

```

Comment:
Hey Megumi, 
be lazy, use ternary operator whenever you have such a simple if-else case ;)